packages/website/docs/templates.mdx

```diff
@@ -0,0 +1,810 @@
+---
+title: Config templates
+authors:
+  - name: Clément Vannicatte
+    title: Software Engineer @ Algolia
+    url: https://github.com/shortcuts
+---
+
+To help you create the best search experience for your users, we provide config templates for multiple websites generators. If you'd like to add a new template to our list, or believe we should update an existing one, please [send us an email][1] or [open a pull request][2]
```

Comment:
I would emphasis that all of them are accesible in the Crawler UI 
And also that they are Crawler config.

A screenshot can help